Makefile

```diff
@@ -337,12 +344,32 @@ ifneq ($(NODEP),1)
 export MACOSX_DEPLOYMENT_TARGET := 11
 endif
 
+# TODO: Find a better way to limit this to SA1!
+ifeq ($(BUILD_NAME), sa1)
+ifeq ($(CC1), $(DEFAULT_CC1))
```

Comment:
Why not just platform == gba? We don't offer the GBA build on modern, (and I don't think we need to)

Reply:
Because oldagbcc doesn't have the compiler flag in it, on purpose.
It's enabled there by default.

Reply:
Not sure I understand?

Reply:
Does this line not do that job?
`$(C_BUILDDIR)/lib/m4a/m4a.o: PROLOGUE_FIX :=`